### ISSUE TITLE
Not working; showing invalid host name

### ISSUE THREAD

**Issue by oreki89** [amraymanaly/kissdownloader#2]
Hello. I tried your script but its not working. Here's what is shows:
./kissdownloader -s http://kissasian.com/Drama/Kekkon-Dekinai-Otoko 1 -q 480p
Fetching http://kissasian.com/Drama/Kekkon-Dekinai-Otoko ...
Fetching http://kissasian.com/Drama/Kekkon-Dekinai-Otoko/Episode-1?id=18124 ...
Using default quality ()
Downloading Kekkon Dekinai Otoko - Episode 1 ..

http://: Invalid host name.

**Comment by amraymanaly** [comment]
Yeah they updated the site, so my script is now obsolete. I'll be fixing it soon. Hopefully.
